hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -199,6 +200,20 @@
       en_run_modes: ["sw_test_mode"]
       run_opts: ["+use_spi_load_bootstrap=1"]
     }
+    {
+      name: chip_sw_uart_baudrate_256k
+      uvm_test_seq: chip_sw_uart_tx_rx_vseq
+      sw_images: ["sw/device/tests/uart_tx_rx_test:1"]
+      en_run_modes: ["sw_test_mode"]
+      run_opts: ["+uart_baud_rate=256000"]
```

Comment:
@tjaychen @eunchan I'm testing 1m, 1.5m, 256k for uart baud rate and sys clk is 24M. Is there any other frequency we should test in top-level?